common/primitives/src/schema.rs

```diff
@@ -1,11 +1,12 @@
-#[cfg(feature = "std")]
 use crate::utils;
 use codec::{Decode, Encode};
 use scale_info::TypeInfo;
 #[cfg(feature = "std")]
 use serde::{Deserialize, Serialize};
 use sp_std::prelude::*;
 #[cfg(feature = "std")]
+use types::ModelType;
```

Comment:
I don't think this is correct. Since `#[cfg(feature = "std")]` is on top of `use types::ModelType;` this include will only happen in `std` builds and it will cause issues in run time build.

I think lines [8-10] should probably be like the following
```
#[cfg(feature = "std")]
use utils::*;
use types::ModelType;
```

Reply:
Yes you are correct, I'll pay closer attention to the cfg macros above use statements